fetch.bs

```diff
@@ -2277,6 +2277,10 @@ otherwise, it is unset.
 <a for=request>done flag</a>, and <a for=request>timing allow failed flag</a> are used as
 bookkeeping details by the <a for=/>fetch</a> algorithm.
 
+<p>A <a for=/>request</a> has an associated boolean
+<dfn export for=request>text directive user activation</dfn>.
+Unless stated otherwise, it is false.
```

Comment:
We should group this with "user-activation". I'm not entirely sure anymore why that has a hyphen in it either. Hopefully @sideshowbarker remembers.

And do we really need both? Seems unfortunate.

Reply:
I don't remember, but regardless I would imagine "user-activation" should only have a hyphen in cases where it's used as an adjective. If it's used as a noun, we shouldn't need any hyphen.

Reply:
> And do we really need both? Seems unfortunate.

Good question. I don't know off the top of my head, I will investigate.